tests/integration/linux_bridge_test.py

```diff
@@ -64,14 +67,33 @@
 
 @pytest.fixture
 def bridge0_with_port0(port0_up):
+    with _bridge0_with_port0(port0_up) as state:
+        yield state
+
+
+@pytest.fixture
+def bridge0_with_port0_uses_port_mac_explicitly(port0_up):
```

Comment:
The fixture name `uses_port_mac_explictly` does not match the fixture but comes from the test. Maybe use something like `bridge0_with_port0_with_explicit_port_mac`?